src/components/internal/OverlayTrigger.tsx

```diff
@@ -35,6 +35,7 @@ export interface OverlayTriggerProps {
   /** Prop set the style of the button element */
   variant?: ButtonVariant;
   hideEndAdornment?: boolean;
+  showActiveBorder?: boolean;
```

Comment:
**thought:** Since we define the `showActiveBorder` prop here, I would expect that it impacts any variation of the OverlayTrigger, which means it could be a text button, Icon button, or avatar button.